shell/platform/common/cpp/public/flutter_texture_registrar.h

```diff
@@ -23,7 +23,9 @@ typedef struct FlutterDesktopTextureRegistrar*
 // Additional types may be added in the future.
 typedef enum {
   // A Pixel buffer-based texture.
-  kFlutterDesktopPixelBufferTexture
+  kFlutterDesktopPixelBufferTexture,
+  kFlutterDesktopGpuBufferTexture,
+  kFlutterDesktopTextureNone
```

Comment:
What's purpose of this?

Reply:
we need add a type for gpu buffer data：
1.Plugin sends FlutterDesktopTextureInfo to flutter engine.
`int64_t TextureRegistrarImpl::RegisterTexture(TextureVariant* texture) {
  if (auto pixel_buffer_texture = std::get_if<PixelBufferTexture>(texture)) {
    FlutterDesktopTextureInfo info = {};
    info.type = kFlutterDesktopPixelBufferTexture;
    info.pixel_buffer_config.user_data = pixel_buffer_texture;
    info.pixel_buffer_config.callback =
        [](size_t width, size_t height,
           void* user_data) -> const FlutterDesktopPixelBuffer* {
      auto texture = static_cast<PixelBufferTexture*>(user_data);
      auto buffer = texture->CopyPixelBuffer(width, height);
      return buffer;
    };

    int64_t texture_id = FlutterDesktopTextureRegistrarRegisterExternalTexture(
        texture_registrar_ref_, &info);
    return texture_id;
  } else if (auto gpu_buffer_texture = std::get_if<GpuBufferTexture>(texture)) {
    FlutterDesktopTextureInfo info = {};
    info.type = kFlutterDesktopGpuBufferTexture;
    info.pixel_buffer_config.user_data = gpu_buffer_texture;
    info.pixel_buffer_config.gbCallback =
        [](size_t width, size_t height,
           void* user_data) -> const FlutterDesktopGpuBuffer* {
      auto texture = static_cast<GpuBufferTexture*>(user_data);
      auto buffer = texture->GetGpuBuffer(width, height);
      return buffer;
    };`
2.flutter engine will create different ExternalTexture based on FlutterDesktopTextureType(class FlutterTizenTextureRegistrar)

Reply:
Thanks for explaining :smile: 
And.. ,actually, My intention is ``kFlutterDesktopTextureNone``

Reply:
I find we don't need kFlutterDesktopTextureNone,  I have remove it.